demos/guide/chain.pem

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We seem to be dynamically creating the chain.pem file - but we've also added it to the source tree. Was that intentional?

Reply:
Ah, I thought it was in master too, but I guess it's not. I'll remove it from the source tree